### PR TITLE
update docs on how to add the dependency to the tqg framework

### PR DESCRIPTION
**Pull request by DeromirNeves**
Resolves tq1/br-tqg-ios#145

### CODE REVIEW

File: docs/menus/1-Installation.md [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+        config.build_settings.delete 'CODE_SIGN_IDENTITY[sdk=iphoneos*]'
+        config.build_settings['EXPANDED_CODE_SIGN_IDENTITY'] = ""
+        config.build_settings['CODE_SIGNING_REQUIRED'] = "NO"
+        config.build_settings['CODE_SIGNING_ALLOWED'] = "NO"
+      end
+    end
+  end
+end
+```
+0. Run pod install
```
Comment (askmon) [inline docs/menus/1-Installation.md]: This is being show as 1 on Markdown, should be 5.

File: docs/menus/1-Installation.md [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+
+Possible build errors:
+
+0. If you receive an error such as: 'TQTStylesheets.h' file not found. 
+You must, after adding use_frameworks!, change all your imports that refer to Pods to the following format, to read from the framework instead of static library: `<Pod_Name/Header_filename.h>`
+For instance:
+Change: `#import “TQTStylesheets.h”`
+To: `#import <TQTStylesheets/TQTStylesheets.h>`
+NOTE: It doesn't apply for the TQ1SDK Pod, since this Pod uses a static library. So just use, for exemple: `#import <TQ1Inbox.h>`
```
Comment (askmon) [inline docs/menus/1-Installation.md]: "exemple"

File: docs/menus/1-Installation.md [hunk 3]
```diff
@@ -1,6 +1,48 @@
...
+This error also happens if you use "link_with" in your Podfile. Since "link_with" was introduced only in Pods 0.39, and will be discontinued in Pods 1.0.0 (replaced by abstract_target), the solution is to just organize your Podfile without using "link_with".
+
+0. Everything compiles, but when you run the project it crashes with the error:
+`dyld: Library not loaded: @rpath/libswiftContacts.dylib
+  Referenced from: /Users/raphaelpetegrosso/Library/Developer/CoreSimulator/Devices/11321430-86AF-41AD-8231-E13E5245497D/data/Containers/Bundle/Application/BD8B1C09-8DEC-4A22-B64D-BABEC45C9547/TQGTest.app/Frameworks/libswiftUIKit.dylib
+  Reason: image not found`
+To solve this, open your build settings and search for the entry "Embedded Content Contains Swift Code". Set it to YES. By setting this option, you add some dylib needed by Swift to your build, such as "libswiftUIKit.dylib", shown in the error message.
+More information here: https://developer.apple.com/library/ios/qa/qa1881/_index.html
+NOTE: This problem was reported with when using cocoapods 0.39.0.beta.4, but didn't happen for 0.39.0.
```
Comment (askmon) [inline docs/menus/1-Installation.md]: "with when"

Comment (DeromirNeves) [inline docs/menus/1-Installation.md]: updated

File: docs/menus/1-Installation.md (hunk 2 above)
Comment (DeromirNeves) [inline docs/menus/1-Installation.md]: updated

File: docs/menus/1-Installation.md (hunk 1 above)
Comment (DeromirNeves) [inline docs/menus/1-Installation.md]: updated

**Comment by askmon** [comment]
👍

**Pull request closed without merging**